views/js/smaily_module.js

```diff
@@ -81,4 +81,66 @@ $(document).ready(function() {
       }
     });
   });
+  // If autoresponders allready validated call smaily api to populate autoresponders list
+  (() => {
```

Comment:
ES6 syntax does not work in older browsers. Is Prestashop not supported by older browsers and usage of ES6 is OK?

Reply:
Changed function to ES5 syntax.
d0b3e76d32877f3fe09afb0e0a1e5a83639205b1

Reply:
Isn't `let` also part of ES6 syntax?

Reply:
Went trough previous JS file and switched all to `var ` @kaittodesk